kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/conversation/fragment/MobiComConversationFragment.java

```diff
@@ -1674,6 +1674,9 @@ public void onCreateOptionsMenu(Menu menu, MenuInflater inflater) {
         menu.findItem(R.id.muteGroup).setVisible(false);
         menu.findItem(R.id.unmuteGroup).setVisible(false);
         menu.findItem(R.id.deleteConversation).setVisible(false);
+        if(alCustomizationSettings.isLogoutOptionFromConversation()){
+            menu.findItem(R.id.logout).setVisible(true);
```

Comment:
this if statement if not required, you can optimize this line

Reply:
done as suggested